frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -231,25 +235,10 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>([
                 order,
             }
 
-            if (active !== undefined) {
-                pageFiltersFromUrl.active = String(active)
-            }
+            pageFiltersFromUrl.active = active ? String(active) : undefined
+            pageFiltersFromUrl.page = page ? parseInt(page) : undefined
 
-            if (page !== undefined) {
-                pageFiltersFromUrl.page = parseInt(page)
-            }
-
-            // Initialize filters with the URL params if none are set
-            const isInitializingFilters =
-                objectsEqual(DEFAULT_FILTERS, values.filters) && !objectsEqual(DEFAULT_FILTERS, pageFiltersFromUrl)
-            /**
-             * Pagination search param in the URL is modified directly by the LemonTable component,
-             * so let's update filter state if it changes
-             */
-            const isChangingPage = page !== undefined && page !== values.filters.page
-            if (isInitializingFilters || isChangingPage) {
```

Comment:
These checks were previously preventing a loop, but aren't needed; the new diff 
```
setActiveTab: () => {
            // Don't carry over pagination from previous tab
            actions.setFeatureFlagsFilters({ page: 1 }, true)
        },
```
prevents this loop by resetting the filters properly. Not having this check ensures we (properly) reload the table whenever we navigate back to the features list